pymatgen/analysis/tests/test_phase_diagram.py

```diff
@@ -90,30 +91,37 @@ def test_str(self):
     def test_read_csv(self):
         entries = EntrySet.from_csv(str(module_dir / "pdentries_test.csv"))
         self.assertEqual(entries.chemsys, {"Li", "Fe", "O"}, "Wrong elements!")
-        self.assertEqual(len(entries), 492, "Wrong number of entries!")
+        self.assertEqual(len(entries), 490, "Wrong number of entries!")
 
 
 class TransformedPDEntryTest(unittest.TestCase):
     def setUp(self):
+        # NOTE a 1:1 isn't the best test? maybe a tertiary system should be used.
         comp = Composition("LiFeO2")
         entry = PDEntry(comp, 53)
-        self.transformed_entry = TransformedPDEntry(
-            {DummySpecies("Xa"): 1, DummySpecies("Xb"): 1}, entry
-        )
+
+        terminal_compositions = ["LiO", "FeO"]
+        terminal_compositions = [Composition(c) for c in terminal_compositions]
+
+        sp_mapping = OrderedDict()
+        for i, comp in enumerate(terminal_compositions):
+            sp_mapping[comp] = DummySpecies("X" + chr(102 + i))
+
```

Comment:
For this code block, why not just:

```
sp_mapping = {
    Composition("LiO"): DummySpecies("Xa"),
    Composition("FeO"): DummySpecies("Xb")
}
```

Reply:
This is how the sp_mapping is setup automatically so I just copied it over here. It starts from 102 which is `f `rather than 97 which is `a` which is why I start from `f`.